code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -1171,8 +1171,7 @@ func (sa *StorageAllocation) validateEachBlobber(
 }
 
 // Until returns allocation expiration.
-func (sa *StorageAllocation) Until() common.Timestamp {
-	var conf = Config{}
+func (sa *StorageAllocation) Until(conf *Config) common.Timestamp {
```

Comment:
Pass the MaxChallengeCompletionTime as param should be enough as this method doesn't care about the other configs.

Reply:
Made the changes